about/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
```

Comment:
Why your file index.html in the root directory is empty?

Reply:
Fixed error in tag <html lang="ru">